scripts/pipelines/github/templates/test/test-pipeline.yml.template

```
@@ -0,0 +1,48 @@
+name: $pipelineName
+
+on:
+  workflow_run:
+    workflows: $buildPipelineName
+    types: completed
```

with the ci_pipeline we need to change this trigger
```suggestion
  workflow_call:
```